bindings_ffi/src/mls.rs

```diff
@@ -142,7 +140,6 @@ pub async fn create_client(
 #[allow(unused)]
 #[uniffi::export(async_runtime = "tokio")]
 pub async fn get_inbox_id_for_address(
-    logger: Box<dyn FfiLogger>,
```

Comment:
How will this work now for surfacing libxmtp errors in native? https://github.com/xmtp/xmtp-android/blob/main/library/src/main/java/org/xmtp/android/library/libxmtp/XMTPLogger.kt

Reply:
Ahh sorry just read you description so it should just work now with out this! Nice

Reply:
Yup just tested with android, we actually get way more info now: 

![Screenshot 2024-12-04 at 11 47 46 AM](https://github.com/user-attachments/assets/d30105b2-fac2-4d95-863b-66604db0f8a1)